pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 120
+target_version = ['py37']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+  | foo.py           # also separately exclude a file named foo.py in
+                     # the root of the project
+)
+'''
```

Comment:
This looks useless, using black is accepting black, not trying to adapt it.
So if you stay at `88` columns intesad of 120 (my screen barely fit 88 already when veritically split) you just don't need the file at all, just run `black potodo` and no need for include or excludes or nothing.